ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTestFuns.scala

```diff
@@ -136,6 +136,8 @@ trait AbstractHttpServiceIntegrationTestFuns
   import HttpServiceTestFixture._
   import json.JsonProtocol._
 
+  override protected lazy val devMode: Boolean = true
```

Comment:
I would rather add a TODO here to ensure testing with both upgradable and non-upgradable LF versions, instead of permanently switching on `devMode`